static/src/javascripts/projects/common/modules/experiments/join-discussion-after-poll.js

```diff
@@ -54,7 +54,7 @@ define([
         var messageElem = document.createElement('div');
 
         messageElem.classList.add('join-discussion-after-poll');
-        messageElem.innerHTML = '<p>Thanks for taking part! Why not <a href="#comments">join the discussion</a>?</p>';
+        messageElem.innerHTML = '<p>Thanks for taking part! Why not <a data-link-name="Join discussion" href="#comments">join the discussion</a>?</p>';
```

Comment:
I would make this slightly more descriptive - `poll: cta: join discussion` so it's less likely to conflict with another event